v1/topdown/tokens.go

```diff
@@ -1226,7 +1264,61 @@ func getInputSHA(input []byte, h func() hash.Hash) []byte {
 	return hasher.Sum(nil)
 }
 
+type jwtCacheEntry struct {
+	payload        ast.Object
+	header         ast.Object
+	validSignature bool
+}
+
+const tokenCacheName = "io_jwt"
```

Comment:
In the future, this will configurable? Or we would expose this to the user in some way so they know what name to use in the config?

Reply:
The cache name itself isn't configurable no, and I don't think there is a use for that. The `io_jwt` name is exposed in the documentation for how to configure the jwt cache.